config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: ghcr.io/kedacore/keda-olm-operator
+  newName: quay.io/jvaldes/kedacore/keda-olm-operator
```

Comment:
This looks like stuff that snuck in from your testing. Is that right?

Reply:
Yikes! Removed.